cephfs/path_test.go

```diff
@@ -60,3 +60,273 @@ func TestRemoveDir(t *testing.T) {
 	assert.EqualError(t, err,
 		fmt.Sprintf("stat %s: no such file or directory", localPath))
 }
+
+func TestLink(t *testing.T) {
+	mount := fsConnect(t)
+	defer fsDisconnect(t, mount)
+
+	t.Run("directory operations", func(t *testing.T) {
```

Comment:
Unless there's a really good reason not to, please use camelCase names for the sub-tests. Go will auto-convert the spaces, but I think it is preferable to have the names in the code match the test output exactly when we can. This applies to the other t.Run calls below as well.

Reply:
Done